Marlin/src/module/planner.h

```diff
@@ -240,11 +245,17 @@ typedef struct PlannerBlock {
   uint32_t accelerate_before,               // The index of the step event where cruising starts
            decelerate_start;                // The index of the step event on which to start decelerating
 
-  #if ENABLED(S_CURVE_ACCELERATION)
+  #if ENABLED(SMOOTH_LIN_ADV)
+    uint32_t cruise_time;
```

Comment:
Needs a comment. I think the correct comment would be "Cruise time in STEP timer counts".